<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="carName">
          <property name="text">
           <string>Car Name: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="carEngine">
          <property name="text">
           <string>Car Engine: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="carBonnet">
          <property name="text">
           <string>Car Bonnet:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="carPrice">
          <property name="text">
           <string>Car Price:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="carName_2">
          <property name="text">
           <string>Car Name: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="carEngine_2">
          <property name="text">
           <string>Car Engine: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="carBonnet_2">
          <property name="text">
           <string>Car Bonnet:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="carPrice_2">
          <property name="text">
           <string>Car Price:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="carName_3">
          <property name="text">
           <string>Car Name: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="carEngine_3">
          <property name="text">
           <string>Car Engine: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="carBonnet_3">
          <property name="text">
           <string>Car Bonnet:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="carPrice_3">
          <property name="text">
           <string>Car Price:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
